libcontainer/configs/config.go

```diff
@@ -83,6 +83,8 @@ type Syscall struct {
 	Args     []*Arg `json:"args"`
 }
 
+type Capabilities = specs.LinuxCapabilities
```

Comment:
Should mark the local `Capabilities` as deprecated ("use `specs.LinuxCapabilities` instead")?

Reply:
Hmm; perhaps we can just remove the type altogether.